lib/generamba/dsl/methods.rb

```diff
@@ -0,0 +1,15 @@
+module Generamba
+  module DSL
+    module Methods
```

Comment:
What do you think of adding logging capability? Something like `log(message, log_level)`, where each `log_level` has a corresponding text color.